themes/theme-ios11/themeApi/ProductCard/components/Render/index.jsx

```diff
@@ -67,9 +68,11 @@ function ProductCardRender({
 
       {(!(hidePrice && hideRating)) && (
         <div className={style}>
-          {(!hideRating && rating && rating.average > 0) && (
+          {!hideRating && rating && rating.average > 0 ? (
             <RatingStars value={product.rating.average} />
-          )}
+          ) :
+            !hideRating && showEmptyRatingStars && product.rating &&
+            <RatingStars value={product.rating.average} />}
```

Comment:
please don't nest ternary expressions. Instead you could determine the expression before the return statement and then render the RatingStars component depending on this result.